validation/cross_validate_rust_python.py

```
@@ -104,9 +104,14 @@ def carreau_yasuda_python(shear_rate):
     print(f"  Type: CarreauYasudaBlood")
     print(f"  Method: apparent_viscosity(shear_rate)")
     
-    # Try to test if we can create a blood model
-    # Note: This depends on cfd_python API structure
-    print(f"\n  TODO: Add cfd_python API test if blood model is exposed")
+    print(f"\n  Cross-checking Python vs Rust:")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)